async-http-client/src/main/scala/org/http4s/asynchttpclient/client/AsyncHttpClient.scala

```diff
@@ -15,8 +15,8 @@
  */
 
 package org.http4s
-package client
 package asynchttpclient
+package client
```

Comment:
This one's a little weird because `client` is part of the name and it appears twice, but at least it's consistent.

Reply:
Yeah, I noticed it too. :/